installing/installing_aws/installing-aws-secret-region.adoc

```diff
@@ -0,0 +1,72 @@
+[id="installing-aws-secret-region"]
+= Installing a cluster on AWS into a Top Secret Region
+include::modules/common-attributes.adoc[]
+:context: installing-aws-secret-region
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Amazon Web Services (AWS) into a Commercial Cloud Services (C2S) Top Secret Region. To configure the
+region, modify parameters in the `install-config.yaml` file before you
+install the cluster.
+
+== Prerequisites
```

Comment:
Anchor!

see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Done. Thank you.